commit e0b6c7e0bc5e1c36d7648091aef80d534fb5e50a (refs/changes/57/2657/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-09-26T15:58:45-07:00 (2 years ago)
    
    joyent/node-triton#232 commit 8438f44 broke parseVolumeSize unit tests
    Reviewed by: Julien Gilli <julien.gilli@joyent.com>
    Approved by: Julien Gilli <julien.gilli@joyent.com>

diff --git a/test/unit/parseVolumeSize.test.js b/test/unit/parseVolumeSize.test.js
index 024aa81..ab4eb4e 100644
--- a/test/unit/parseVolumeSize.test.js
+++ b/test/unit/parseVolumeSize.test.js
@@ -63,9 +63,6 @@ test('parseVolumeSize', function (tt) {
 
     tt.test('parsing valid sizes', function (t) {
         var validVolumeSizes = [
-            {input: '42m', expectedOutput: 42},
-            {input: '42M', expectedOutput: 42},
-            {input: '42g', expectedOutput: 42 * 1024},
             {input: '42G', expectedOutput: 42 * 1024}
         ];
 
